pai-management/paictl.py

```diff
@@ -163,26 +165,38 @@ def cluster_object_model_generate_service(config_path):
 
 
 
-def hadoop_ai_build(os_type = "ubuntu16.04"):
+def pai_build():
 
     parser = argparse.ArgumentParser()
     parser.add_argument('-p', '--path', required=True, help="The path of your configuration directory.")
```

Comment:
>path [](start = 33, length = 4)

"--config-path" for more meamingful full name?

Reply:
resolved.